app/nginx-1.11.10/src/event/ngx_event.c

```diff
@@ -239,7 +251,19 @@ ngx_process_events_and_timers(ngx_cycle_t *cycle)
 
     delta = ngx_current_msec;
 
-    (void) ngx_process_events(cycle, timer, flags);
+    //(void) ngx_process_events(cycle, timer, flags);
```

Comment:
Why `ngx_process_events` is commented?

Reply:
I comment it for comparison test, sorry that you see it. 
Should I close this PR, and create a new one?